src/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -119,11 +119,17 @@ void CommandLine::DumpHelp(const char* program)
     printf("     Ignored: neither MSVCDIS nor CoreDisTools is available.\n");
 #endif // USE_COREDISTOOLS
     printf("\n");
+    printf(" -forcejitoption key=value\n");
+    printf("     Set the JIT option named \"key\" to \"value\" for JIT 1. Overwrites the existing value if it was. NOTE: do not use a \"COMPlus_\" prefix!\n");
+    printf("\n");
```

Comment:
"Overwrites the existing value if it was" -- this is ambiguous (to me). Do you mean, "Overwrites the existing value if it was already set"? If so, please update this (and forcejit2option)

Reply:
Yes, fixed.